sites/website/src/docs/design-systems/overview.md

```diff
@@ -0,0 +1,64 @@
+---
+id: overview
+title: What is the Design System?
+sidebar_label: Overview
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/design-systems/overview.md
+---
+A *Design System* is somewhat of an amorphous term but can generally be thought of as a collection of resources for interactive media that promotes brand alignment. While that definition is intentionally broad, in UI development, Design Systems generally manifest as component libraries surrounded by usage guidance and design principles.
+
+In code, FAST currently uses "Design System" in a narrower sense than the above; the Design System is a dictionary of data that informs the visual (and sometimes functional) representation of UI elements. The Design System captures, stores, and reflects any information that should inform the rendering of a UI element. This data is often referred to as *Design Tokens*. Common examples of data in a Design System are:
+- UI Colors
+- Fonts and type ramps
+- Motion data such as timings and easing curves
+- UI Density
+- Spacing values
+
+## How the Design System is used
+There are two primary ways that Design System values (Design Tokens) are used:
+1. In CSS stylesheets
+2. Programmatically in JavaScript
+
+### Design System in CSS
+Many Design Tokens are used as CSS property values in component stylesheets. Assume for a moment that a Design Token `font-size-large` exists in a Design System, you may see something like the following:
+
+```css
+:host {
+    font-size: var(--font-size-large);
+}
+```
+
+### Design System in JavaScript
+It is sometimes the case that the Design System must be used in JavaScript. Assume for a moment that a Design Token `fadeInDuration` exists in a Design System, you may see something like the following:
```

Comment:
a succinct way of saying the same thing may be:  "For Design Systems used in JavaScript, you may see a Design Token `fadeInDuration` shown as:" You could do the same above as well "For Design Systems used in CSS, you may see a Design Token `font-size-large` shown as:"

Reply:
Adjusting to be more succinct